components/sample-rhdevs-website/EventPopup.tsx

```diff
@@ -0,0 +1,84 @@
+import React from 'react'
+import { useState } from 'react'
+import styled from 'styled-components'
+import SignUpButton from './SignUpButton'
+
+const EventPopupContainer = styled.div`
+  position: absolute;
+  background-color: white;
+  padding: 20px;
+  margin-top: 100px;
+  margin-left: 20%;
+  margin-right: 20%;
+`
+
+const CloseButton = styled.button`
+  border-radius: 50%;
+  display: flex;
+  margin-left: auto;
+  background-color: #595959;
+  color: white;
+  font-weight: 600;
+  &:hover {
+    cursor: pointer;
+  }
+`
+
+const PopupImage = styled.img`
+  display: block;
+  border-radius: 10px;
+  max-height: 70%;
+  max-width: 70%;
+  margin: 0 auto;
+  padding-top: 30px;
+  padding-bottom: 50px;
+`
+
+const PopupTitle = styled.h1`
+  display: flex;
+  font-size: 24px;
+  font-weight: 600;
+  color: black;
+`
+const PopupContent = styled.p`
+  white-space: pre-line;
+`
+
+type Props = {
+  image: any
```

Comment:
Avoid the use of the type `any` here. Judging from the use, it seemed to be a string

Reply:
```suggestion
  image: string
```
yeap, since it's a URL, it's a string. Try to avoid using `any`